src/main/resources/messages_en_US.properties

```diff
@@ -821,6 +821,7 @@ pdfOrganiser.mode.6=Odd-Even Split
 pdfOrganiser.mode.7=Remove First
 pdfOrganiser.mode.8=Remove Last
 pdfOrganiser.mode.9=Remove First and Last
+pdfOrganiser.mode.10=Odd-Even Merge
```

Comment:
please add to GB file not US for the base

Reply:
Added also to the ``messages_en_GB.properties``